### PR TITLE
new testimonial changes

### PR DESCRIPTION
**Pull request by matinmolkara**

### CODE REVIEW

File: src/app/globals.css
Comment (vali-mohammadi) [inline src/app/globals.css:1]: Can we customize swiper without using the global css file? I think there must be a better solution that they suggest.

File: src/app/components/home/testimonial/index.tsx [hunk 1]
```diff
@@ -1,29 +1,56 @@
...
             className="size-60 mb-12 md:mb-0 bg-white  border-primary-light border rounded-3xl p-2"
           >
             <div className="mb-0">
               <div className="">
                 <Image
+             
                   width="50"
                   height="50"
-                  src={c.profile_picture || ""}
+                  src={c.author.profile_picture || ""}
```
Comment (vali-mohammadi) [inline src/app/components/home/testimonial/index.tsx:53]: let's have a default profile image for the cases that we dont have a profile picture (It's a best practice but you can ignore it as we will never fall into that situation)

**Review by kave-me — CHANGES_REQUESTED** [review]

File: src/app/globals.css
Comment (matinmolkara) [reply src/app/globals.css:1]: unfortunately no, i think easy way is the best way !

File: src/app/components/home/testimonial/index.tsx (hunk 1 above)
Comment (matinmolkara) [reply src/app/components/home/testimonial/index.tsx:53]: definitely, i will try that

**Pull request merged**